contrib/kafka/samples/README.md

```diff
@@ -8,6 +8,10 @@ sink.
 This sample demonstrates how to configure, deploy, and use the Apache Kafka
 Event Source with a Knative Service.
 
+For kubernetes, a simple Kafka installation can be done with Strimzi, check out their
```

Comment:
all references must be "Apache Kafka", because that's ASF trademark policy

Reply:
As with above, will fix in a follow up commit.